CMakeLists.txt

```diff
@@ -172,13 +168,14 @@ if (KAGEN_USE_CGAL)
     set(CGAL_DO_NOT_WARN_ABOUT_CMAKE_BUILD_TYPE TRUE CACHE BOOL "Do not warn about Debug mode")
     set(CGAL_DONT_OVERRIDE_CMAKE_FLAGS TRUE CACHE BOOL "Force CGAL to maintain CMAKE flags")
     find_package(CGAL QUIET)
+
     if (CGAL_FOUND)
-      add_definitions(-DKAGEN_CGAL_FOUND)
-      if (${CGAL_VERSION} VERSION_GREATER_EQUAL 5)
-	list(APPEND KAGEN_LINK_LIBRARIES CGAL::CGAL)
-      else()
-        include(${CGAL_USE_FILE})
-      endif()
+        add_definitions(-DKAGEN_CGAL_FOUND)
+        if (${CGAL_VERSION} VERSION_GREATER_EQUAL 5)
+            list(APPEND KAGEN_LINK_LIBRARIES CGAL::CGAL)
+        else()
+            include(${CGAL_USE_FILE})
+        endif()
     else ()
         message(STATUS "Could not find the CGAL library: Random Delaunay Graphs will not be available")
```

Comment:
set `KAGEN_USE_CGAL` to `OFF` here, because otherwise `KagenConfig.cmake` still searches for CGAL and fails if it is missing.